css/_uswds-theme-custom-styles.scss

```diff
@@ -1568,47 +1568,55 @@ iframe {
   margin-bottom: units(4);
 }
 
-.post-excerpt {
-  margin-top: units(5);
+.post-preview {
```

Comment:
**issue**: We have a `custom-styles` with partials. Can we create a partial for post styles there?

Reply:
Added a `posts` partial in [fece742](https://github.com/uswds/uswds-site/pull/2979/commits/fece742770fae7746ec2b82dee4b598702ceeae6)